src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<blockquote>The models we present in the cell below are very simple and the results they give are <strong>approximative</strong>.</blockquote><p>--&gt; approximate</p><p> </p><blockquote>All main functions and sub-functions are stored in <a href="https://github.com/fast-aircraft-design/FAST-OAD/raw/cf8fa880a6d979a72ead2cc6fe2c5613b7bd453d/src/fastoad/notebooks/02_explanation/modules/pure_python" target="_blank">the modules folder</a> </blockquote><p>The link does not work. Not sure there is a way to make it work. Better to put it as plain text. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
> --> approximate

Corrected in 3ba48f01


> The link does not work in Jupyter Lab. Not sure there is a way to make it work. Better to put it as plain text.

Proposed correction is using tooltips to keep the folders highlighted while not having unusable links. Implemented in 3ba48f01

Reply:
<p>That was worth a try, but I think that any user will conclude the links are just bugged. And the benefit of the tooltips is small.</p><p>I think we should rather linking only to files and put things like that:</p><p> </p><p>"""</p><p>The <strong>geometry</strong> folder contains:</p><ul><li><a href="http://localhost:8888/files/src/fastoad/notebooks/02_explanation/modules/pure_python/geometry/sub_components/compute_wing_area.py?_xsrf=2%7Ca8ec4d77%7Cf24490cd6290add0809427b224851e60%7C1655298951" target="_blank">The wing area computation</a></li></ul><p>"""</p><p> </p><p>Before this list, you may simply tell that these files are in <code>modules/pure_python</code>.</p>